src/options/generate/ui.py

```diff
@@ -0,0 +1,82 @@
+import random
+from dataclasses import dataclass
+
+product_manager_names = [
+    ('Leon', 'm'),
+    ('Saahil', 'm',),
+    ('Susana', 'f')
+]
+engineer_names = [
+    ('Joschka', 'm'),
+    ('Johannes', 'm'),
+    ('Johanna', 'f'),
```

Comment:
we have biased sampling but I don't mind 😅

Reply:
nice catch!
I used the three examples to let gpt4 generate the rest.
I removed them now